app/src/main/java/com/capstone/event_finder/fragments/FeedFragment.java

```diff
@@ -18,25 +17,14 @@
 
 import com.capstone.event_finder.R;
 import com.capstone.event_finder.activities.ErrorActivity;
-import com.capstone.event_finder.activities.MainActivity;
 import com.capstone.event_finder.adapters.EventsAdapter;
-import com.capstone.event_finder.interfaces.EventFetcherInterface;
 import com.capstone.event_finder.models.Event;
 import com.capstone.event_finder.network.EventViewModel;
-import com.capstone.event_finder.network.RetrofitClient;
-import com.google.gson.JsonArray;
-import com.google.gson.JsonObject;
-import com.parse.ParseUser;
 
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.List;
 
-import retrofit2.Call;
-import retrofit2.Callback;
-import retrofit2.Response;
-
-public class FeedFragment extends Fragment implements EventFetcherInterface {
+public class FeedFragment extends Fragment {
```

Comment:
✅ This all looks good as far as the `FeedFragment` goes.  

You can see how much simpler this fragment has become, now that it only handles UI-related code as [Android's documentation suggests](https://developer.android.com/topic/architecture)